trace-obfuscation/src/replacer.rs

```diff
@@ -12,7 +12,13 @@ struct RawReplaceRule {
     repl: String,
 }
 
-#[derive(Debug)]
+impl PartialEq for ReplaceRule {
+    fn eq(&self, other: &Self) -> bool {
+        self.name == other.name && self.repl == other.repl && self.re.as_str() == other.re.as_str()
```

Comment:
Can't derive Eq/PartialEq because rust devs say that two regular expressions should be equal if their state machines are equal, not just the literal string. That's a level deeper than we need, so this is sufficient.